benjaminkott/bootstrap_package
# [TASK] adjust typoscript constants for social media links so that the…

# Pull Request

## Related Issues

* Relates #1298

## Prerequisites


## Description

Adjust typoscript constants for social media links so that the label are editable in the backend

## Steps to Validate

1. Go to TypoScript module in backend and switch to Constant Editor
2. Select category "bootstrap package: social media"
3. Edit a label (for which a url is given)
4. Go to frontend and check that the changed label is displayed on mouseover
